src/main/java/tasks/EventTask.java

```diff
@@ -0,0 +1,129 @@
+package tasks;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * The EventTask Class inherits Task Class
+ * to support different specificities of a task
+ * as per input by user.
+ */
+public final class EventTask extends Task {
+    /**
+     * Stores the type of task information
+     * that identifies a EventTask.
+     */
+    private final String taskType = "[E]";
```

Comment:
Should Javadoc of class members be specified on a single line?

Reply:
changed to single line, thanks for catching my mistake!